ui/jobs/jobs_config.ts

```diff
@@ -237,6 +239,11 @@ UserConfig.registerOptions('jobs', {
           'Wenn der Countdown 0 erreicht.': 'expired',
           'Wenn der Countdown nahe 0 ist.': 'threshold',
         },
+        ko: {
+          '비활성화': 'disabled',
+          '카운트 다운이 0초일 때': 'expired',
+          '카운트 다운이 0초에 가까워 질때': 'threshold',
```

Comment:
I think "0초에 가깝다" is not very clear in this sentence, maybe translate it as "when it reaches the threshold(閾値)" just like [me and Echoring do](https://github.com/quisquous/cactbot/pull/3151/files) would be more accurate?

Reply:
I think threshold value would change by each jobs? I thought if there's no exact time written, it would rather better write ambiguous word. People may try to find what exact threshold is, from cactbot readme or somewhere.

Reply:
Yeah, that's true.
Then I thought maybe "when the counter is almost ending" would be another choice?
FYI, which is called "終了間際" in Japanese or "即將結束" in Chinese.

Reply:
"when the counter is almost ending" is better I agree, I'll see if there's better one.

Reply:
By the way, some thresholds may be a little long...

Reply:
I understood, I have Korean translation in mind.